mirdata/download_utils.py

```diff
@@ -71,14 +75,27 @@ def downloader(
     if not os.path.exists(save_dir):
         os.makedirs(save_dir)
 
+    if not index:
```

Comment:
Does this mean that we have to specify the index in the donwload function?

Reply:
Yes, now it's a required argument of the downloader. It could be optional, but I made it mandatory for now to avoid future bugs in case we update an indexes to be remote - this way we don't have to remember to also update the downloader.

Reply:
from the conversation offline I understood that the `initialize` function is the one to set the index version, and that propagates to the rest of the code. That is smooth from the point of view of the user. Did I get it right?

Reply:
yes exactly